app/assets/src/components/views/amr_heatmap/AMRHeatmapView.jsx

```diff
@@ -185,10 +206,31 @@ export default class AMRHeatmapView extends React.Component {
     }
   }
 
+  getDownloadCSVLink() {
+    const csvOption = DOWNLOAD_OPTIONS[0];
```

Comment:
I don't think you need `DOWNLOAD_OPTIONS`. You can replace `{csvOption.text}` below with the string value.